zerver/openapi/zulip.yaml

```diff
@@ -7807,6 +7842,31 @@ paths:
                               type: integer
                               description: |
                                 The ID of the video chat provider.
+                      realm_giphy_rating_options:
```

Comment:
I don't really understand why you have this and a duplicate `giphy_rating_options` above.  

I would have expected us to just have one copy of the `options` object in the API documentation, and for it to be called `giphy_rating_options`, since this is settings data that is not realm-dependant.

Reply:
Ah! I think I was doing this late night and probably confused things.